kostyor/tests/unit/test_db_api.py

```diff
@@ -57,21 +57,40 @@ def test_get_all_clusters(self):
 
         expected = db_api.get_clusters()
 
-        self.assertEqual(len(expected['clusters']), 10)
+        self.assertEqual(len(expected['clusters']), 11)
```

Comment:
This test was kind of brittle, maybe we should just change it to assertGreaterThan 0 ?